packages/js/product-editor/src/hooks/use-product-manager/use-product-manager.ts

```diff
@@ -70,17 +70,21 @@ export function useProductManager< T = Product >( postType: string ) {
 			setIsSaving( true );
 
 			await validate( extraProps );
+			const { saveEntityRecord } = dispatch( 'core' );
 
-			// @ts-expect-error There are no types for this.
-			const { editEntityRecord, saveEditedEntityRecord } =
-				dispatch( 'core' );
-
-			await editEntityRecord< T >( 'postType', postType, id, extraProps );
+			const { blocks, content, selection, ...editedProduct } = WPSelect(
```

Comment:
I think the WPSelect alias should starts with lower case since it refers to a function a not to a type, interface, class or component name.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/45436/commits/de14c1a86615adbc75270e1a38ee33c721fa8b90